app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntryEffect.kt

```diff
@@ -60,14 +60,22 @@ data class CreateNewBpEntry(
     val patientUuid: UUID,
     val systolic: Int,
     val diastolic: Int,
-    val date: LocalDate
-) : BloodPressureEntryEffect()
+    val parsedDateFromForm: LocalDate,
+    val prefilledDate: LocalDate
+) : BloodPressureEntryEffect() {
+  val wasDateChanged: Boolean
+    get() = parsedDateFromForm != prefilledDate
```

Comment:
`parsedDateFromForm` was a little hard to read since both `from` and `form` are anagrams and right next to each other. Can we rename it to something else like `userEnteredDate` or something?